<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DPT Pulse Generator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Circuit Parameters Input</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Bus Voltage(Vbus):</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtVbus">
     <property name="geometry">
      <rect>
       <x>149</x>
       <y>30</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>400</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>214</x>
       <y>33</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>V</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Inductor Current:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>214</x>
       <y>63</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>A</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtIndCur">
     <property name="geometry">
      <rect>
       <x>149</x>
       <y>60</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>160</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Load Inductor:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>214</x>
       <y>93</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>uH</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtLoadInd">
     <property name="geometry">
      <rect>
       <x>149</x>
       <y>90</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblWhlPulse">
     <property name="geometry">
      <rect>
       <x>243</x>
       <y>15</y>
       <width>161</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resource.qrc">:/Icons/WholePulse.PNG</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>661</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Time Parameters Input</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>400</width>
       <height>231</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <property name="rowCount">
      <number>4</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>132</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>35</number>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>Parameters</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="background">
       <color>
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Default Values</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="background">
       <color>
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Set Values</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>T1 Pluse width</string>
      </property>
      <property name="flags">
       <set>ItemIsEnabled</set>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>40 (us)</string>
      </property>
      <property name="flags">
       <set>ItemIsEnabled</set>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>40</string>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>T2 Pluse width</string>
      </property>
      <property name="flags">
       <set>ItemIsEnabled</set>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>10 (us)</string>
      </property>
      <property name="flags">
       <set>ItemIsEnabled</set>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>10</string>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>T3 Pluse width</string>
      </property>
      <property name="flags">
       <set>ItemIsEnabled</set>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>10 (us)</string>
      </property>
      <property name="flags">
       <set>ItemIsEnabled</set>
      </property>
     </item>
     <item row="2" column="2">
      <property name="text">
       <string>10</string>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string>TBreak</string>
      </property>
      <property name="flags">
       <set>ItemIsEnabled</set>
      </property>
     </item>
     <item row="3" column="1">
      <property name="text">
       <string>1 (Sec)</string>
      </property>
      <property name="flags">
       <set>ItemIsEnabled</set>
      </property>
     </item>
     <item row="3" column="2">
      <property name="text">
       <string>1</string>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btnCopy">
     <property name="geometry">
      <rect>
       <x>292</x>
       <y>200</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Copy Default</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>231</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="rad1Pulse">
      <property name="geometry">
       <rect>
        <x>119</x>
        <y>22</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>1 Pluse</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>136</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Resource.qrc">:/Icons/xPluse.PNG</pixmap>
      </property>
     </widget>
     <widget class="QRadioButton" name="rad8Pulse">
      <property name="geometry">
       <rect>
        <x>119</x>
        <y>142</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>8 Pluses</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="rad2Pulse">
      <property name="geometry">
       <rect>
        <x>119</x>
        <y>60</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>2 Pluses</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="rad5Pulse">
      <property name="geometry">
       <rect>
        <x>119</x>
        <y>100</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>5 Pluses</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>16</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Resource.qrc">:/Icons/1Pulse.PNG</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>56</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Resource.qrc">:/Icons/2Pulse.PNG</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>96</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Resource.qrc">:/Icons/xPluse.PNG</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnGo">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>210</y>
       <width>81</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>GO</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkRepeat">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>220</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Repetitive Run</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Board Connection</string>
    </property>
    <widget class="QLabel" name="lblConnect">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Designed by Rex</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resource.qrc">:/Icons/Inactive.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
